packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -163,11 +163,12 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
   const strings = { ...localeStrings, ...props.strings };
 
   const avatarOptions = {
-    text: displayName,
+    text: displayName ?? strings.displayNamePlaceholder,
```

Comment:
On the participant list in the ui snapshots we are still seeing UP instead of the person tile image we are seeing in the video gallery. Is this expected?

Reply:
Was still working on this PR. It is fixed now.